dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -112,7 +113,7 @@ public QuayImageRegistry(final Token quayToken) {
     }
 
     //TODO: If the repo has a lot of tags, then it needs to be paged through. Can get tag info individually, but then that's more API calls.
-    private Set<Image> getImagesForTag(final Tool tool, final Tag tag, final QuayTag quayTag) {
+    private Optional<Image> getImageForTag(final Tool tool, final Tag tag, final QuayTag quayTag) {
```

Comment:
Comment above seems to be out-of-date now.
I guess the bigger issue is I don't understand how/if this deals with paging and the identified problem that we may need to page through a large number of tags in order to get images.

Reply:
yeah, that is in another ticket https://ucsc-cgl.atlassian.net/browse/DOCK-1211